OracleSOASuite/dockerfiles/12.2.1.4/README.md

```diff
@@ -1,82 +1,186 @@
 SOA on Docker
 =============
 
-Sample Docker configurations to facilitate installation, configuration, and environment setup for Docker users. This project includes quick start dockerfiles for Oracle SOA 12.2.1.4.0 based on Oracle Linux 7, Oracle JRE 8 (Server) and Oracle Fusion Middleware Infrastructure 12.2.1.4.0.
+Sample Docker configurations to facilitate installation, configuration, and environment setup for Docker users. This project includes quick start dockerfiles for Oracle SOA 12.2.1.4.0 based on Oracle Linux 7, Oracle JRE 8 (Server), and Oracle Fusion Middleware Infrastructure 12.2.1.4.0.
 
-At the end of this configuration there may be 3 containers running: 
-1. Oracle Database Container (Optional: only when RCU schema is created in a database running in a container) 
-2. Oracle Weblogic Administration Server Container 
-3. Oracle Weblogic Managed Server Container (Oracle SOA Server or Oracle Service Bus Server)
+At the end of this configuration there may be 4 containers running:
+1. Oracle Database Container (optional: only when RCU schema is created in a database running in a container)
```

Comment:
I would put optional at the beginning, i.e. "1. (Optional) Oracle Database container"